src/client/mongodb/repository/BaseMongooseRepositoryImpl.ts

```
@@ -155,7 +155,7 @@
       const result = await this.model.insertMany(documents, { lean: true });
       return result as any[];
     } catch (error) {
-      throw new Error('Error inserting documents');
+      throw error;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)